src/tracks/SampleTrack.cpp

```diff
@@ -835,6 +858,15 @@ SampleTrackView::SampleTrackView( SampleTrack * _t, TrackContainerView* tcv ) :
 
 
 
+void SampleTrackView::updateIndicator()
+{
+	if (model()->isPlaying()) m_activityIndicator->activateOnce();
```

Comment:
```suggestion
	if (model()->isPlaying()) { m_activityIndicator->activateOnce(); }
```

Not specifically needed, but always good to have.